### PR TITLE
API Management Control Plane Swagger - Direct Route - First Version

### PR DESCRIPTION
**Pull request by solankisamir**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, see this page.
- [x]  Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x]  If applicable, the PR references the bug/issue that it fixes.
- [x]  Swagger files are correctly named (e.g. the api-version in the path should match the api-version in the spec).

Quality of Swagger
- [x] I have read the contribution guidelines.
- [x]  My spec meets the review criteria
- [x]  The spec conforms to the Swagger 2.0 specification.
- [x]  The spec follows the guidelines described in the Swagger checklist document.
- [ ]  Validation tools were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/apimanagement/data-plane/readme.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- Microsoft.ApiManagement/preview/2017-03-01/apimanagement.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimapis.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimauthorizationservers.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimbackends.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimcertificates.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimemailtemplate.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimgroups.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimidentityprovider.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimloggers.json
+- Microsoft.ApiManagement/preview/2017-03-01/apimnetworkstatus.json
```
Comment (sergey-shandar) [inline specification/apimanagement/data-plane/readme.md]: 'Microsoft.ApiManagement/preview/2017-03-01/apimnetworkstatus.json' is missing.

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

File: specification/apimanagement/data-plane/readme.md (hunk 1 above)
Comment (solankisamir) [reply specification/apimanagement/data-plane/readme.md]: #Fixed. Good catch.

**Comment by solankisamir** [comment]
@sergey-shandar ping on this.

File: specification/apimanagement/data-plane/Microsoft.ApiManagement/preview/2017-03-01/apimquotas.json [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+          "type": "string",
+          "format": "date-time",
+          "description": "The date of the start of Counter Period. The date conforms to the following format: `yyyy-MM-ddTHH:mm:ssZ` as specified by the ISO 8601 standard.\n"
+        },
+        "periodEndTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The date of the end of Counter Period. The date conforms to the following format: `yyyy-MM-ddTHH:mm:ssZ` as specified by the ISO 8601 standard.\n"
+        },
+        "Value": {
```
Comment (sergey-shandar) [inline specification/apimanagement/data-plane/Microsoft.ApiManagement/preview/2017-03-01/apimquotas.json]: The name `Value` should follow camelCase style, for example `value`.

**Review by sergey-shandar — CHANGES_REQUESTED** [review]
Please, provide examples to all operations.

File: specification/apimanagement/data-plane/Microsoft.ApiManagement/preview/2017-03-01/apimquotas.json (hunk 2 above)
Comment (solankisamir) [reply specification/apimanagement/data-plane/Microsoft.ApiManagement/preview/2017-03-01/apimquotas.json]: This issue is being fixed in next api-version https://github.com/Azure/azure-rest-api-specs/issues/1987

**Comment by sergey-shandar** [comment]
@solankisamir Could you provide examples for the specs?

**Comment by solankisamir** [comment]
@sergey-Shandar working on examples

**Comment by solankisamir** [comment]
closing the pull request and opening a new one with multiple changes.

**Comment by sergey-shandar** [comment]
@solankisamir please, add a link to the new PR so we can track and understand context. It will also help reviewers to see if it's partially reviewed.

**Pull request closed without merging**